src/main/resources/com/mathworks/ci/BuildArtifactAction/index.jelly

```diff
@@ -82,7 +81,12 @@
                                     </j:if>
                                 </j:if>
                                 <j:if test="${p.taskSkipped == true}">
-                                    <font color="#FCE94F"> SKIPPED </font>
+                                    <font color="#FCE94F">
+                                        SKIPPED
+                                       <j:if test="${p.skipReason != ''}">
```

Comment:
Good catch there on indentation, fixed it. For casing, I'll wait for @mw-hrastega's suggestion

Reply:
Yes, it's good to be consistent with how other text elements are presented. We don't seem to use all caps elsewhere. (If you meant to show property values, then you needed to use a different font instead of capitalization: https://www.mathworks.com/help/matlab/ref/matlab.buildtool.taskresult-class.html#mw_446baf3c-50f2-4885-b93e-73e93c8644cb. However, I think you show Passed (?) in certain cases, which is not a property value.)

Reply:
Updated, thanks!